packer/scylla_install_image

```diff
@@ -114,37 +115,45 @@ if __name__ == '__main__':
         # disable unattended-upgrades
         run('apt-get purge -y unattended-upgrades update-notifier-common')
 
-    # use easy_install instead of pip, because easy_install requires
-    # fewer dependencies, no need to install gcc, it makes our AMI smaller.
-    if distro == 'centos':
-        run('yum install -y epel-release')
-        run('yum install -y pystache python-daemon python-setuptools')
-        # Default install prefix on CentOS7 is /usr not /usr/local, but we need
-        # to align path for executable, so specify script-dir to /usr/local/bin
-        run('python -m easy_install --script-dir /usr/local/bin https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz')
-        pyver = ''
-    else:
-        # XXX: we want install dependencies from distro repo, but
-        # python-daemon-2.2 on Ubuntu 20.04 is too new for aws-cfn-bootstrap,
-        # so install python-daemon-2.1 from pypi
-        run('apt-get install -y python3-pystache python3-lockfile python3-setuptools')
-        run('python3 -m easy_install https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-py3-latest.tar.gz')
-        pyver = '3'
-    cfnpath = out('python{} -c "import cfnbootstrap as _; print(_.__path__[0])"'.format(pyver))
-    initpath = os.path.abspath('{}/../init/{}/cfn-hup'.format(cfnpath, 'redhat' if distro == 'centos' else 'ubuntu'))
-    shutil.copyfile(initpath, '/etc/init.d/cfn-hup')
-    os.chmod('/etc/init.d/cfn-hup', 0o755)
+    if args.target_cloud == 'aws':
+        # use easy_install instead of pip, because easy_install requires
+        # fewer dependencies, no need to install gcc, it makes our AMI smaller.
+        if distro == 'centos':
```

Comment:
We don't support CentOS based AMIs anymore. I think we can just drop this option.

Reply:
@hagitsegev Can we just drop CentOS7 based AMI from next branch? If so, scripts become much simpler.
But I think it should be separated commit, separated PR, since it will be large changes
(we can drop half of scripts, drop scylla-machine-image rpm, drop support python2 in scylla_install_* script... etc)

Reply:
@syuu1228 I am not sure we should drop it yet, just today @hagitsegev had to build AMI based on Centos. let's keep it for a while and then do the cleanup